Packs/AWS-EC2/ReleaseNotes/1_3_0.md

```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### New: AwsEC2GetAccounts
```

Comment:
GetAccounts sounds like a read and we're doing an update here. We need to think of a better name.